reviewing-prs.md

```diff
@@ -91,9 +91,11 @@ This is the detailed code review of the actual changes, covering:
 
 Some code style guidelines can be found in the [Flink Code Style Page]({{ site.baseurl }}/contribute-code.html#code-style)
 
-## Review Template
+## Review with the @flinkbot
 
-The "Review Progress" section is posted at the end of the description of every new pull request.
+The Flink community is using a service called [@flinkbot](https://github.com/flinkbot) to help with the review of the pull requests.
+
+The bot automatically posts a comment tracking the review progress for each new pull request:
```

Comment:
I assume the bot will also cover flink-web and flink-shaded?

Reply:
No :( The current implementation only supports one repository.
I would propose to activate the bot on the apache/flink repo to fine-tune and debug it. Once we are happy with it, I can extend it to support multiple repositories.